docs/user-guide/rest-api-cluster-tags.md

```diff
@@ -0,0 +1,38 @@
+# REST-API - Cluster Tags
+
+The `tags` property in cluster object is a comma-separated list of tags that are associated to the cluster. Each tag is a free-text string that can consist of the following characters:
+Alphanumeric (aA-zZ, 0-9), underscore (_) and white-spaces.
```

Comment:
how do we remove tags? is an empty string a valid value?

Reply:
right, it can be cleared by specifying an empty string, added a note for that.